test/sql/size_utils.sql

```diff
@@ -164,11 +164,49 @@ SELECT * FROM approximate_row_count();
 SELECT * FROM approximate_row_count(NULL);
 \set ON_ERROR_STOP 1
 
-SELECT * FROM chunks_detailed_size(NULL);
-SELECT * FROM hypertable_detailed_size(NULL);
+-- Test size functions with NULL input
+SELECT * FROM hypertable_size(NULL);
```

Comment:
can you add a test for hypertable_size(0) and hypertable_size(1)

Reply:
Did you add tests for this? I don't see them.

Reply:
I missed it, but added now.